packages/foam-vscode/src/features/document-link-provider.spec.ts

```diff
@@ -10,16 +10,22 @@ import {
 import { LinkProvider } from './document-link-provider';
 import { OPEN_COMMAND } from './utility-commands';
 import { toVsCodeUri } from '../utils/vsc-utils';
+import { updateFoamVsCodeConfig } from '../services/config';
 
 describe('Document links provider', () => {
   const parser = createMarkdownParser([]);
 
   beforeAll(async () => {
     await cleanWorkspace();
+    await updateFoamVsCodeConfig('edit.linkReferenceDefinitions', 'off');
   });
 
   afterAll(async () => {
     await cleanWorkspace();
+    await updateFoamVsCodeConfig(
+      'edit.linkReferenceDefinitions',
+      'withoutExtensions'
```

Comment:
if what you are trying to achieve is removing the config set above I believe the best approach is using `undefined` here, see docs for `vscode.workspace.getConfiguration().update()`